### PR TITLE
[ZARAZ] Added info regarding debug mode

### PR DESCRIPTION
**Pull request by marciocloudflare**

### CODE REVIEW

File: products/zaraz/src/content/datalayer-compatibility.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ order: 3
 pcx-content-type: reference
 ---
 
-# dataLayer compatibility mode
+# Data Layer compatibility mode
```
Comment (pedrosousa) [inline products/zaraz/src/content/datalayer-compatibility.md]: Not sure if "Layer" should be capitalized here, if we're making this change.

File: products/zaraz/src/content/debug-mode.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+order: 4
+pcx-content-type: how-to
+---
+
+# Debug mode
+
+Zaraz offers a debug mode to troubleshoot the events and triggers systems. To activate the debug mode you need to create a special debug cookie using your debug key.
```
Comment (pedrosousa) [inline products/zaraz/src/content/debug-mode.md]:
```suggestion
Zaraz offers a debug mode to troubleshoot the events and triggers systems. To activate the debug mode you need to create a special debug cookie containing your debug key. 
```

File: products/zaraz/src/content/debug-mode.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# Debug mode
+
+Zaraz offers a debug mode to troubleshoot the events and triggers systems. To activate the debug mode you need to create a special debug cookie using your debug key. 
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account and website.
+1. Go to **Zaraz**. 
+1. In the main Zaraz screen, click **Settings**.
+1. Copy your **Debug Key**.
+1. Open a web browser and access its Developer Tools. For example, to access Developer Tools in Google Chrome click **View** > **Developer** > **Developer Tools**.
```
Comment (pedrosousa) [inline products/zaraz/src/content/debug-mode.md]:
```suggestion
1. Open a web browser and access its Developer Tools. For example, to access Developer Tools in Google Chrome, click **View** > **Developer** > **Developer Tools**.  
```

File: products/zaraz/src/content/debug-mode.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# Debug mode
+
+Zaraz offers a debug mode to troubleshoot the events and triggers systems. To activate the debug mode you need to create a special debug cookie using your debug key. 
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account and website.
+1. Go to **Zaraz**. 
+1. In the main Zaraz screen, click **Settings**.
+1. Copy your **Debug Key**.
+1. Open a web browser and access its Developer Tools. For example, to access Developer Tools in Google Chrome click **View** > **Developer** > **Developer Tools**.  
+1. Click the **Console** pane and write the following command to create a debug cookie:
```
Comment (pedrosousa) [inline products/zaraz/src/content/debug-mode.md]:
```suggestion
1. Click the **Console** pane and enter the following command to create a debug cookie: 
```

File: products/zaraz/src/content/debug-mode.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+1. Go to **Zaraz**. 
+1. In the main Zaraz screen, click **Settings**.
+1. Copy your **Debug Key**.
+1. Open a web browser and access its Developer Tools. For example, to access Developer Tools in Google Chrome click **View** > **Developer** > **Developer Tools**.  
+1. Click the **Console** pane and write the following command to create a debug cookie: 
+
+  ```txt
+  document.cookie="debug=YOUR_DEBUG_KEY"
+  ```
+1. Refresh your browser to see the debug logs. They will appear in the Console pane.
```
Comment (pedrosousa) [inline products/zaraz/src/content/debug-mode.md:21]:
```suggestion

1. Refresh your browser to see the debug logs. They will appear in the Console pane.
```
Adding an empty line.

File: products/zaraz/src/content/debug-mode.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+Zaraz’s debug mode is now enabled. However, due to the temporary nature of the Console pane in web browsers, you need to activate the **Preserve Log** option for the Console pane. If you do not activate this option, you will lose the previous debug information whenever the Console refreshes.
+
+To activate the Preserve Log option in Google Chrome, click the **settings** cog > **Preserve logs**.
```
Comment (pedrosousa) [inline products/zaraz/src/content/debug-mode.md]:
```suggestion
To activate the **Preserve Log** option in Google Chrome, click the **settings** cog > **Preserve logs**.
```

**Review by pedrosousa — APPROVED** [review]

File: products/zaraz/src/content/datalayer-compatibility.md (hunk 1 above)
Comment (marciocloudflare) [reply products/zaraz/src/content/datalayer-compatibility.md]: The more I think about this, the more I think it should be lowercase.

**Pull request merged**